examples/client/client.c

```diff
@@ -1598,8 +1598,8 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
 
 #ifdef HAVE_SNI
     if (sniHostName)
-        if (wolfSSL_CTX_UseSNI(ctx, 0, sniHostName, XSTRLEN(sniHostName))
-                                                               != SSL_SUCCESS) {
+        if (wolfSSL_CTX_UseSNI(ctx, 0, sniHostName,
+                    (unsigned short) XSTRLEN(sniHostName)) != SSL_SUCCESS) {
```

Comment:
Please use `word16` instead of `unsigned short`.

Reply:
✔️